ex10_html-css-fundamentals-part2

Indigoinspn opened the pull request:
task-01.html  - Form with different input types. Created according to given design.
task-02.html  - Table created according to given design.
task-03.html  - Adaptive landing page created according to given design.

Indigoinspn committed:
ex10_html-css-fundamentals-part2
task-01.html  - Form with different input types. Created according to given design.
task-02.html  - Table created according to given design.
task-03.html  - Adaptive landing page created according to given design.

Indigoinspn committed:
Replace  % units in margin/padding properties with rem units (task-01.html,task-02.html)

Indigoinspn committed:
Remove some empty lines (task-03.js, task-03.css)

Indigoinspn committed:
Remove some empty lines (task-03.html)

rakotoz commented on src/ex10_html-css-fundamentals-part2/task-01.html:
```diff
@@ -0,0 +1,189 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Form</title>
+    <style>
```
Не используй инлайн стили

Indigoinspn replied:
Вынесла в отдельный файл.

rakotoz commented on src/ex10_html-css-fundamentals-part2/task-02.html:
```diff
@@ -0,0 +1,124 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Table</title>
+    <style>
```
не используй инлайн стили

Indigoinspn replied:
Вынесла в отдельный файл.

rakotoz commented on src/ex10_html-css-fundamentals-part2/task-02.html:
```diff
@@ -0,0 +1,124 @@
+            font-size: 0.625rem;
+        }
+
+        table {
```
не используй тэги в css

rakotoz commented on src/ex10_html-css-fundamentals-part2/task-03.css:
```diff
@@ -0,0 +1,467 @@
+/**********************  NAVIGATION BAR  ************************/
+
+/************ Gamburger and dropdown menu ***********/
+#gamburger {
```
Не используй id в css

rakotoz commented on src/ex10_html-css-fundamentals-part2/task-03.css:
```diff
@@ -0,0 +1,467 @@
+    }
+
+
+    #tooltip {         
```
не появляется при наведении

Indigoinspn replied:
у меня работает по клику) 
![image](https://user-images.githubusercontent.com/43875213/58099646-bde25f00-7be4-11e9-99de-d7a9d248c4d6.png)

Indigoinspn replied:
Переделала на ховер.

rakotoz commented on src/ex10_html-css-fundamentals-part2/task-03.css:
```diff
@@ -0,0 +1,467 @@
+    height: 10.7rem;    
+    margin:20% 60% 30%;    
+    color:#ffffff;    
+    border: 1px solid #bfc5c9;     
```
Рамки быть не должно

Indigoinspn replied:
да, убрала.

rakotoz commented on src/ex10_html-css-fundamentals-part2/task-03.html:
```diff
@@ -0,0 +1,139 @@
+                </div>
+                <!--****************** Slider ******************-->
+                <div class="slider">
+                    <img src="slider1.jpg" alt="Slide 1">
```
Это должна быть не картинка, а сверстаный блок с текстом и картинкой

Indigoinspn replied:
С каким текстом? не совсем тебя поняла

rakotoz replied:
![68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6d656469612d702e736c69642e65732f75706c6f6164732f3635393433302f696d616765732f343239383030372f6375747465642e6a7067  1711×1350 ](https://user-images.githubusercontent.com/16106071/58100185-d141fa00-7be5-11e9-8de7-2bb276bd2afb.png)

Indigoinspn replied:
Поправлю

Indigoinspn replied:
Доработала, теперь можно поставить/снять "лайк", также реагирует на ховер.

rakotoz commented on src/ex10_html-css-fundamentals-part2/task-03.css:
```diff
@@ -0,0 +1,467 @@
+
+/*************** MEDIA QUERIES common rules for large devices ****************/
+
+@media only screen and  (min-width: 1200px) {
```
Вынеси медиа в отдельный файл и проверь мобилку все поехало

Indigoinspn replied:
пришли скрин плз... у меня все ок было

Indigoinspn replied:
Мобильной версии не было в задании, только desctop и tablet. Думаю если есть недоработки то это не критично.

rakotoz replied:
![Untitled](https://user-images.githubusercontent.com/16106071/58100443-4f9e9c00-7be6-11e9-86c5-4f77157a9a88.png)

Indigoinspn replied:
и что поехало? для мобилки не было макета, на мой взгляд все норм)  или ты про текст под лампой , то что на синий фон залез?  А то что надписи в центре заехали.... ну на макете тоже есть наложение... разве плохо смотрится, был бы макет сделала бы по макету, а это мое видение)?

Indigoinspn replied:
Исправила. Лампа не наезжает на синий фон если уменьшать ширину экрана вплоть до 430px.

Indigoinspn replied:
Медиа правила вынесла в отдельный файл.

rakotoz replied:
отлично)

rakotoz commented on src/ex10_html-css-fundamentals-part2/task-03.js:
```diff
@@ -0,0 +1,54 @@
+var gamburger = document.getElementById("gamburger-icon");
+var dropdownMenu = document.getElementById("dropdown-menu");
+
+function showDropdownMenu(){
```
А как закрыть выпадающее меню, не кликая по пунктам?

Indigoinspn replied:
Никак пока, выпадающее меню это было опционально, т.ч. пока в сыром варианте.

rakotoz replied:
А зачем тогда залила, если оно не работает? Просто это выглядит не очень

Indigoinspn replied:
Мы это еще не проходили на тот момент ( я про события), сделала как могла... знаний не хватало на тот момент, но я постаралась сделать больше, чем было в задании, пойти дальше!

Indigoinspn replied:
Сделала toggle функцию

Indigoinspn replied:
Также меню скрывается, если вывести мышку за границы меню. (для компьютера)

rakotoz replied:
Супер

rakotoz commented on src/ex10_html-css-fundamentals-part2/task-03.js:
```diff
@@ -0,0 +1,54 @@
+var searchField = document.getElementById("search-field");
+var searchIcon = document.getElementById("search-icon");
+
+function showSearchForm(){
```
А как скрыть строку поиска?

Indigoinspn replied:
Это тоже было опционально, тоже пока в сыром варианте.

rakotoz replied:
А зачем тогда залила, если оно не работает? Просто это выглядит не очень

Indigoinspn replied:
Сделала toggle функцию

rakotoz commented on src/ex10_html-css-fundamentals-part2/task-03.js:
```diff
@@ -0,0 +1,54 @@
+var geoIcon = document.getElementById("geo");
+var popUp = document.getElementById("tooltip");
+
+function showGeoLocation() {
```
Это должно происходить при наведении мыши на иконку и не только на одну, а на все

Indigoinspn replied:
С чего ты взял?

rakotoz replied:
Тултипы всегда показываются при наведении. Это всплывающая подсказка при показе. Но, ок. Оставь как есть

Indigoinspn replied:
Хорошо)

Indigoinspn replied:
Исправила.

Indigoinspn committed:
Move css rules to the separate file (task-01, task-02)
Task-03
Change event for contact's icons from click to hover;
Remove frame for central part content;
Redo slider: add "like icon and text; add hover and click events;
Move media rules to separate file.
Add extra space between central part and articles for mobile view;
Create toggle function for gamburger menu and search-field for mobile
view.
